examples/processing/centering_the_zero_beam.py

```diff
@@ -23,7 +23,7 @@
 # In many instances the zero beam position will vary systematically with the scan position.
 # This can be corrected by fitting a linear plane to the zero beam position using the
 # :meth:`make_linear_plane` method.
-shifts.make_linear_plane()  # Making a linear plane to remove the systematic shift
```

Comment:
The functionality is changed here so that the plane is returned. So should AFAIK be `shifts = shifts.make_linear_plane()`